ga4gh/client.py

```diff
@@ -99,8 +103,11 @@ def _deserializeResponse(self, response, protocolResponseClass):
         jsonResponseString = response.text
         self._updateBytesRead(jsonResponseString)
         self._debugResponse(jsonResponseString)
-        responseObject = protocolResponseClass.fromJsonString(
-            jsonResponseString)
+        try:
+            responseObject = protocolResponseClass.fromJsonString(
+                jsonResponseString)
+        except protocol.UnableToDecodeJsonException:
```

Comment:
Are these always the same thing? What happens if the server sends back malformed JSON? I don't really see how re-raising a different exception is helping us here.